index.html

```diff
@@ -26,7 +26,7 @@ <h1>Dev Delta Jokes</h1>
         <li>Martin Placeholder</li>
         <li>Nathan Placeholder</li>
         <li>Sam Placeholder</li>
-        <li>Sean Placeholder</li>
+        <li><a href="sean/joke.html">Sean's Joke Page</a></li> 
```

Comment:
Ayyyy I was gonna make a comment mentioning that you forgot this section but you done guuud

Reply:
Thanks dad!

Reply:
![bitmoji](https://sdk.bitmoji.com/render/panel/46d9e3e9-e019-4a50-a121-7c141743c9d5-4ffd6aca-6fea-46b5-b783-001de5495dd9-v1.png?transparent=1&palette=1&width=246)